source/core/replica-set-high-availability.txt

```diff
@@ -15,29 +19,244 @@ Replica Set High Availability
    :depth: 1
    :class: singlecol
 
-:term:`Replica sets <replica set>` provide high availability using
-automatic :term:`failover`. Failover allows a :term:`secondary` member
-to become :term:`primary` if the current primary becomes unavailable.
+:term:`Replica sets <replica set>` use elections to select the 
+:term:`primary` for the cluster. Replica set elections support
+high availability by allowing clusters to automatically recover 
+normal operations after a primary failover event. Applications can
+continue to interact with the cluster during elections, mitigating
```

Comment:
AFAIK the only interactions allowed are read operations.  (I may also not be thinking of something.)  If this is the case, I'd change "interact with" to "read from".  I don't think we should hide the fact that writes don't get processed during the election; that will set incorrect expectations and those will then lead to customers and users getting pissed off and saying our docs are wrong when they have an election and writes stop.  Being able to read during an election still does mitigate the risk of downtime.  Maybe it's also worth adding a note that writes can be queued and/ or retried from the application side to minimize the impact of the election.